ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-stable-openshift-logging-6.1.yaml

```diff
@@ -75,17 +77,18 @@ tests:
             {"name": "amq-streams", "source": "redhat-operators", "channel": "stable", "install_namespace": "openshift-operators","target_namespaces": "", "operator_group": ""}
         ]
       TEST_ADDITIONAL: ""
-      TEST_FILTERS: CPaasrunOnly;CPaasrunBoth;~Disruptive&;~Longduration&
+      TEST_FILTERS: ~Disruptive&;~Longduration&;~Splunk&;~GCL&;~AZWIF&;~AZMonitor&;~PodDisruptionBudget&
       TEST_IMPORTANCE: ""
       TEST_PARALLEL: "3"
       TEST_ROSA_LOGIN_ENV: ""
       TEST_SCENARIOS: sig-openshift-logging
       TEST_TIMEOUT: "15"
     test:
+    - ref: ipi-install-heterogeneous
```

Comment:
It makes me confused, what kind of cluster do you want to install? Heterogeneous or arm64?